src/Uno.UI.Wasm/ts/WindowManager.ts

```diff
@@ -584,6 +584,26 @@ namespace Uno.UI {
 			}
 		}
 
+		public setGeneralStyle(styleElementId: string, cssText: string) {
+			let styleElement = document.getElementById(styleElementId) as HTMLStyleElement;
+			if (!styleElement) {
+				styleElement = document.createElement("style");
+				styleElement.id = styleElementId;
+				styleElement.type = "text/css";
+				document.body.appendChild(styleElement);
+			}
+			styleElement.innerHTML = "";
+			var styleText = document.createTextNode(cssText);
+			styleElement.appendChild(styleText);
```

Comment:
Try to add a class instead.

Reply:
Will do. But in general, do you see some cleaner way to implement this than generating new `<style>` for each `TextBox` and deleting it afterwards? I can't find any alternative...

Reply:
As I said... you can just add a CSS class to the element for that instead of generating new styles dynamically.

Reply:
Im sorry, I'm not sure I understand 🤔. I need the `::selection` pseudoselector to target only the selected text of the input, which seems to require a full CSS rule, as far as I found. If I use a unique class, it makes it easier to target the specific element, but I will still need something like `.myclass::selection` and because each input can have potentially different highlight color, I may need a unique rule for each input. Am I missing something 😅?